tests/console/curl_upload_logs.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary:  Export the existing status of running tasks and system load
+# for future reference
+# Maintainer: Zaoliang Luo <zluo@suse.de>
+
+use base "consoletest";
+use testapi;
+use utils;
+use ipmi_backend_utils 'use_ssh_serial_console';
+use strict;
+
+sub run {
+    my $self = shift;
+    # let's see how it looks at the beginning
+    save_screenshot;
+    check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
+
+    # Export the existing status of running tasks and system load for future reference (fail would export it again)
+    script_run "ps axf > /tmp/psaxf.log";
+    script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup.txt";
+    save_screenshot;
```

Comment:
same here, we do not show the output of the previous commands so the screenshot would not help much